_labs/lab02.markdown

```diff
@@ -32,8 +32,10 @@ reference for most of the LLVM API used in this lab and also throughout the cour
 
 ### Setup
 
-+ Navigate to the `lab2` folder on your computer and open it in VS code.
-+ Press F1 key to display a search bar, search and select `Reopen in Container`.
++ Open the lab2 folder in VS Code, using the terminal command `code <lab2 directory>` 
```

Comment:
put lab2 inside backticks 

can you put the shell command on it's own code block, see how we show shell commands in the coursevm doc for comparison.

Reply:
Made these changes too